server/postcss.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
suggestion (non-blocking): consider adding a comment here explaining what this is doing and why we need it

Reply:
Good idea!  Will add a comment here too.

Reply:
Thanks!